generated/nidcpower/nidcpower/nidcpower_pb2.py

```diff
@@ -15,7 +15,7 @@
 from . import session_pb2 as session__pb2
 
 
```

Comment:
do you know why this changed, and are we sure it's compatbile?

Reply:
The first large section changed because we introduced OUTPUT_STATES_NIDCPOWER_VAL_OUTPUT_INHIBITED to the .proto file.

I think that may be why the serialized start and end values changed, too.

I don't know how it impacts compatibility.
@bkeryan do you know?

Reply:
I do notice that all of the affected values increased by exactly 51.

Reply:
The protobuf compiler parses the `.proto` file and stores it as a protobuf-serialized `FileDescriptorProto` message like this: `_descriptor_pool.Default().AddSerializedFile(b'\n\x0fnidcpower.proto\x12\x0enidcpower_grpc\x1a\x0enidevice.proto\x1a\rsession.proto\"...`

The serialized start/end values are the offsets into that serialized message for each data type.

Adding an enum shifts the serialized offset for every data type declared after it. It's perfectly normal.